en/flight_modes/README.md

```diff
@@ -51,18 +51,18 @@ Throttle Command is mapped direct to Motor Speed.
 
 "Assisted" flight modes are also user controlled but offer some level of "automatic" assistance to gain or restore controlled flight.
 
-- [Altitude](../flight_modes/altitude.md): More easily control vehicle altitude, and in particular reach and maintain a fixed altitude. The mode does not use GPS, and hence will not attempt to hold position/heading against wind.
+- [Altitude](../flight_modes/altitude.md): More easily control vehicle altitude, and in particular reach and maintain a fixed altitude. The mode does not use GPS, and hence will not attempt to hold the x and y position/heading against wind.
 
 - **Position**
-  - **Fixed wing aircraft:** Neutral inputs provide a level flight and
+  - **Fixed wing aircraft:** Neutral inputs (meaning when roll, pitch and yaw sticks are centered) provide a level flight and
     will crab against the wind if needed to maintain a straight line.
   - **Multirotors** Roll controls left-right speed, pitch controls
     front-back speed. When roll and pitch are all centered (inside
     deadzone) the multirotor will hold position. Yaw controls yaw rate
-    as in MANUAL mode. Throttle controls climb/descent rate as in ALTCTL
+    as in MANUAL mode. Throttle controls climb/descent rate as in Altitude
     mode. 
     
-  > **WARNING** Prior to Landing in *Position mode* be sure that landing is 
+  > **Warning** Prior to Landing in *Position mode* be sure that landing is 

```

Comment:
@hamishwillee did you mean lowercase all the letters of "Warning"?

Reply:
No, I meant just like this. We have a gitbook plugin that turns that syntax into a special type of note. Similarly, you can add `> **Note**` and `> **Tip**` for special note rendering.